### PR TITLE
Added unwrap and wrap transformations

### PR DESCRIPTION
**Pull request by davidercruz**
Adds a wrap transformation based on `atoms.wrap` and a unwrap transformation based on @richardjgowers 's `_cutil.make_whole` 

This is the last transformation in my GSOC project :D

PR Checklist
------------
 - [x] Tests?
 - [x] Docs?
 - [x] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

**Comment by jbarnoud** [comment]
Could you rebase against develop? (not merge) The test failure has been fixed in an other pr.

**Comment by jbarnoud** [comment]
Can you tell me what git command you used? It would help me understand what went wrong and how to fix it.

**Comment by davidercruz** [comment]
Here's what I did:
 git remote add upstream https://github.com/MDAnalysis/mdanalysis.git
 git pull --rebase upstream develop

And then solved some conflicts and:

 git rebase --continue

File: testsuite/MDAnalysisTests/transformations/test_rotate.py [hunk 1]
```diff
@@ -150,37 +188,80 @@ def test_rotateby_bad_ag(rotate_universes):
...
     angle = 90
-    vector = [0, 0, 1]
     with pytest.raises(ValueError): 
         rotateby(angle, vector, ag = ag, wrap=True)(ts)
 
-def test_rotateby_bad_center(rotate_universes):
+
+@pytest.mark.parametrize('weights', (
+    " ",
+    "totallynotmasses",
```
Comment (richardjgowers) [inline testsuite/MDAnalysisTests/transformations/test_rotate.py]: 🤣

**Comment by richardjgowers** [comment]
@davidercruz that should have worked, but I guess it hasn't.  Might be best to start a new branch and cherry-pick the commits you made so far.

**Comment by davidercruz** [comment]
@richardjgowers So I should create a new PR and close this one?

**Pull request closed without merging**